be/src/exprs/jit/jit_expr.cpp

```diff
@@ -72,17 +77,21 @@ Status JITExpr::prepare_impl(RuntimeState* state, ExprContext* context) {
         if (!jit_engine->support_jit()) {
             return Status::JitCompileError("JIT is not supported");
         }
-        auto expr_name = _expr->jit_func_name();
+        auto expr_name = _expr->jit_func_name(state);
         _jit_obj_cache = std::make_unique<JitObjectCache>(expr_name, JITEngine::get_instance()->get_func_cache());
 
         auto st = jit_engine->compile_scalar_function(context, _jit_obj_cache.get(), _expr, _children);
-
         auto elapsed = MonotonicNanos() - start;
+        if (state->fragment_ctx() != nullptr) {
```

Comment:
is this judgement necessary? seems state->fragment_ctx() won't be nullptr?

Reply:
in UT tests, it would be nullptr.